rules/manager.go

```diff
@@ -49,7 +45,7 @@ var (
 			Name:      "rule_evaluation_duration_seconds",
 			Help:      "The duration for a rule to execute.",
 		},
-		[]string{ruleTypeLabel},
+		[]string{"rule_type", "status"},
```

Comment:
> If you split out like this PR does people will only graph the success latency, and not understand why their latency sucks when there's lot of timeouts.

I don't understand this, timeouts would be an error and not part of success latency?